drivers/i2c/i2c_ite_enhance.c

```diff
@@ -541,10 +541,6 @@ static int i2c_enhance_pio_transfer(const struct device *dev,
 		data->err = 0;
 		data->msgs = &(msgs[i]);
 
-		if (msgs->flags & I2C_MSG_START) {
```

Comment:
Change line 534 to explicitly set the MSG_START flag on the first message.
current code:
```
	if (data->i2ccs == I2C_CH_NORMAL) {
		msgs->flags |= I2C_MSG_START;
	}
```
change to:
```
	if (data->i2ccs == I2C_CH_NORMAL) {
		&(msgs[0])->flags |= I2C_MSG_START;
	}
```

Reply:
Done, similar changes such as line 938 in i2c_ite_it8xxx2.c.

Reply:
I'm not sure what is the difference...can someone explain that?